hugo/content/3_how-tos/31_account-management/311_managing-projects.md

```diff
@@ -64,6 +64,48 @@ rio project delete <project_name>
 {{% /tab %}}
 {{< /tabs >}}
 
+## Transfer Project Ownership
+
+The project owner can transfer ownership to another member. This solves the issue of restricted modification rights if the original owner leaves the company or is unavailable.
+
+{{% notice info %}}
+If you're part of a project through a user group, you can't become the project owner. You need to be directly added to the project to become an owner.
```

Comment:
A user can be in a user group & have direct access, so I feel the wording here can be approved a little.

Something like this but I think it can be simpler still
"To transfer ownership to a user they must be added to the project directly. Access via user group is not sufficient."